Gems/Atom/RHI/Code/Source/RHI/MultiDeviceSwapChain.cpp

```diff
@@ -85,6 +85,11 @@ namespace AZ::RHI
         return resultCode;
     }
 
+    Ptr<SingleDeviceSwapChain> MultiDeviceSwapChain::GetDeviceSwapChain() const
+    {
+        return AZStd::static_pointer_cast<SingleDeviceSwapChain>(m_deviceObjects.begin()->second);
```

Comment:
Please add a short comment why `m_deviceObjects.begin()` is what we want (since the map only holds one element)